monitoring/ceph-mixin/prometheus_alerts.libsonnet

```diff
@@ -613,7 +613,7 @@
         {
           alert: 'CephPoolNearFull',
           'for': '5m',
-          expr: 'ceph_health_detail{name="POOL_NEAR_FULL"} > 0',
+          expr: 'ceph_health_detail{name="POOL_NEARFULL"} > 0',
```

Comment:
In `OSDMap.cc` we have:

```
    if (!nearfull_detail.empty()) {
      ostringstream ss;
      ss << nearfull_detail.size() << " pool(s) nearfull";
      auto& d = checks->add("POOL_NEARFULL", HEALTH_WARN, ss.str(),
                            nearfull_detail.size());
      d.detail.swap(nearfull_detail);
    }
```

Reply:
Thanks for confirmation. So the change makes sense.  

@anthonyeleven your thoughts?

Reply:
I'm not qualified to make judgement about the presence or absence of the underscore.  I defer to Radek.  My concern was that inconsistency here might result in functional problems.  @rzarzynski is this good to merge?